src/Graphs.jl

```diff
@@ -413,8 +413,11 @@ export
     independent_set,
 
     # vertexcover
-    vertex_cover
+    vertex_cover,
 
+    # planarity
+    is_planar,
+    pmfg
```

Comment:
I would call this `planar_maximally_filtered_graph` -  this is quite a long name, but `pmfg` is not a very common known abbreviation, and using the full name will help with discoverability.

Reply:
Done!